transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -814,39 +814,6 @@ private static void notifyOutboundHandlerException(Throwable cause, ChannelPromi
         PromiseNotificationUtil.tryFailure(promise, cause, promise instanceof VoidChannelPromise ? null : logger);
     }
 
-    private void notifyHandlerException(Throwable cause) {
-        if (inExceptionCaught(cause)) {
-            if (logger.isWarnEnabled()) {
-                logger.warn(
-                        "An exception was thrown by a user handler " +
-                                "while handling an exceptionCaught event", cause);
-            }
-            return;
-        }
-
-        invokeExceptionCaught(cause);
-    }
-
-    private static boolean inExceptionCaught(Throwable cause) {
-        do {
-            StackTraceElement[] trace = cause.getStackTrace();
```

Comment:
Best way to solve the stacktrace inspecting issue :)

Reply:
@slandelle exactly :D